### PR TITLE
Update Cesium, some app tweaks

### PR DESCRIPTION
**Pull request by ggetz**

### CODE REVIEW

File: Source/App.js [hunk 1]
```diff
@@ -26,7 +28,8 @@
     //////////////////////////////////////////////////////////////////////////
 
     // Load Cesium World Terrain
-    viewer.terrainProvider =  Cesium.createWorldTerrain({
+    viewer.terrainProvider = viewer.terrainProvider = new Cesium.CesiumTerrainProvider({
```
Comment (hpinkos) [inline Source/App.js]: You have an extra `viewer.terrainProvider =` here.

And use `Cesium.createWorldTerrain`

File: Source/App.js [hunk 2]
```diff
@@ -110,9 +113,9 @@
                 var latitude = Cesium.Math.toDegrees(cartographicPosition.latitude);
                 var longitude = Cesium.Math.toDegrees(cartographicPosition.longitude);
                 // Modify description
-                var description = '<table class="cesium-infoBox-defaultTable cesium-infoBox-defaultTable-lighter"><tbody>';
-                description += '<tr><th>' + "Latitude" + '</th><td>' + latitude + '</td></tr>';
-                description += '<tr><th>' + "Longitude" + '</th><td>' + longitude + '</td></tr>';
+                description = '<table class="cesium-infoBox-defaultTable cesium-infoBox-defaultTable-lighter"><tbody>';
```
Comment (hpinkos) [inline Source/App.js]: Shouldn't this be `var description`?  I couldn't find the declaration anywhere

File: Source/AppSkeleton.js [hunk 3]
```diff
@@ -1,47 +1,48 @@
...
     // });
     // // Enable depth testing so things behind the terrain disappear.
     // viewer.scene.globe.depthTestAgainstTerrain = true;
 
-    //////////////////////////////////////////////////////////////////////////
-    // Configuring the Scene
-    //////////////////////////////////////////////////////////////////////////
+    // //////////////////////////////////////////////////////////////////////////
+    // // Configuring the Scene
+    // //////////////////////////////////////////////////////////////////////////
```
Comment (hpinkos) [inline Source/AppSkeleton.js]: Revert the extra `//`s

Comment (hpinkos) [reply Source/AppSkeleton.js]: Throughout

**Comment by hpinkos** [comment]
Other than that, it looks good.
Make sure you make the same updates to the tutorial page on cesiumjs.org

File: Source/App.js (hunk 1 above)
Comment (ggetz) [reply Source/App.js]: I copied the sample code from ion here, is `createWorldTerrain` preferred?

Comment (hpinkos) [reply Source/App.js]: yes

File: Source/App.js [hunk 4]
```diff
@@ -110,9 +113,9 @@
...
+                description += '<tr><th>' + "Latitude" + '</th><td>' + latitude.toFixed(5) + '</td></tr>';
```
Comment (hpinkos) [inline Source/App.js]: I commented on the tutorial too, but since Cesium is lon/lat, we should keep Longitude first here

**Comment by ggetz** [comment]
Thanks @hpinkos, updated!

File: Source/App.js [hunk 5]
```diff
@@ -26,7 +28,7 @@
     //////////////////////////////////////////////////////////////////////////
 
     // Load Cesium World Terrain
-    viewer.terrainProvider =  Cesium.createWorldTerrain({
+    viewer.terrainProvider = new Cesium.createWorldTerrain({
```
Comment (hpinkos) [inline Source/App.js]: No `new`, it's just `Cesium.createWorldTerrain`

File: Source/App.js [hunk 6]
```diff
@@ -1,6 +1,8 @@
 (function () {
     "use strict";
 
+    // TODO: Add your ion access token
```
Comment (hpinkos) [inline Source/App.js]: I would to `Cesium.Ion.defaultAccessToken = <insert token from cesium.com here>;` or something like that to give people a little more help

Comment (ggetz) [reply Source/App.js]: I didn't want to break the app right off the bat, but true, I'll have it but comment it out.

**Comment by ggetz** [comment]
@hpinkos Updated, also fixed the tutorial.

**Comment by hpinkos** [comment]
👍

**Pull request merged**